mmaction/models/common/tam.py

```diff
@@ -0,0 +1,133 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import constant_init, kaiming_init, normal_init
+
+
+class TAM(nn.Module):
+    """Temporal Adaptive Module(TAM) for TANet.
+
+    https://arxiv.org/pdf/2005.06803.pdf.
```

Comment:
The list is not in rst form, please refer to https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/backbones/resnet3d_slowfast.py#L366-L367